build.gradle.kts

```diff
@@ -8,7 +8,6 @@ buildscript {
     dependencies {
         classpath(Deps.android_gradle_plugin)
         classpath(Deps.SqlDelight.gradle)
-        classpath(Deps.cocoapodsext)
```

Comment:
Is your cocoapods plugin no longer recommended? If so, do you wanna deprecate/archive it?